src/app/services/xbitapi/xbit-api.service.ts

```diff
@@ -0,0 +1,395 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+import { of } from 'rxjs/observable/of';
+import { GenericHttpService } from '../generichttp/generic-http.service';
+import { getTypeNameForDebugging } from '@angular/core/src/change_detection/differs/iterable_differs';
+
+import { Address } from '../../entities/XBitApi/address';
+import { Administrator } from '../../entities/XBitApi/administrator';
+import { Algorithm } from '../../entities/XBitApi/algorithm';
+import { Balance } from '../../entities/XBitApi/balance';
+import { XBitCoin } from '../../entities/XBitApi/coin';
+import { Country } from '../../entities/XBitApi/country';
+import { Customer } from '../../entities/XBitApi/customer';
+import { Data } from '../../entities/XBitApi/data';
+import { HostingPeriod } from '../../entities/XBitApi/hostingperiod';
+import { Location } from '../../entities/XBitApi/location';
+import { Manufacturer } from '../../entities/XBitApi/manufacturer';
+import { Miner } from '../../entities/XBitApi/miner';
+import { MinerType } from '../../entities/XBitApi/minertype';
+import { MiningFarm } from '../../entities/XBitApi/miningfarm';
+import { UserInformation } from '../../entities/XBitApi/userinformation';
+
+@Injectable()
+
+export class XBitApiService {
+
+    baseUrl = "localhost:9999/api"
+
+    /********* Address *********/
+    // Get Address
```

Comment:
@XBITSwitzerland comments over selfexplanatory functionnames not needed (e.g. "Get Address" comment over "getAddress()" function)